app/assets/javascripts/discourse/app/components/sidebar/edit-navigation-modal-form/categories-form.js

```diff
@@ -13,6 +13,9 @@ export default class extends Component {
   @service siteSettings;
 
   @tracked filter = "";
+  @tracked filteredCategoryIds = new Set();
```

Comment:
Do we need to initialize this new Set here? I think that just `@tracked filteredCategoryIds` will be enough

Reply:
Good point, I've removed that intialization of the set.